app/views/partner_requests/index.html.erb

```diff
@@ -32,6 +35,13 @@
           <tr>
             <th scope="row"><%= link_to partner_request.id, partner_request_path(partner_request) %></th>
             <td><%= partner_request.item_requests.map {|item| "#{item.quantity} of #{item.name}"}.join(", ") %></td>
+            <% if Flipper[:family_requests].enabled?(@partner) %>
+              <td>
+                <% if partner_request.for_families? %>
+                  <%= link_to "Pickup Sheet", family_request_pickup_sheets_path(partner_request) %>
```

Comment:
Probably don't need to address this now, but I wonder if we should eventually make this a bootstrap button with a meaningful icon?

Reply:
Happy to do that now. I'll use the same icon as on the pickup sheets page.

Reply:
Resolved with: 976415f